engine/install/binaries.md

```diff
@@ -173,6 +188,42 @@ The macOS binary includes the Docker client only. It does not include the
     This command downloads a test image and runs it in a container. When the
     container runs, it prints a message and exits.
 
+## Install server and client binaries on Windows
+
+> **Note**
+>
+> The following section describes how to install the Docker daemon on Windows Server which allows you to run Windows containers only. The binaries for Windows do not contain Docker components such as `buildx`, `docker scan`, and `docker compose`. If you are running Windows 10 or 11, we recommend that you install [Docker Desktop](../../desktop/index.md).
```

Comment:
Can you wrap this as well ? Perhaps add `instead` at the end (like we did for macOS);

```suggestion
> The following section describes how to install the Docker daemon on Windows
> Server which allows you to run Windows containers only. The binaries for
> Windows do not contain Docker components such as `buildx`, `docker scan`, and
> `docker compose`. If you are running Windows 10 or 11, we recommend that you
> install [Docker Desktop](../../desktop/index.md) instead.
```